src/Microsoft.DocAsCode.Build.Engine/TemplatePreprocessors/TemplateUtility.cs

```diff
@@ -54,5 +58,33 @@ public string GetHrefFromRoot(string originalHref, string sourceFileKey)
             }
             return file.UrlEncode().ToString() + UriUtility.GetQueryStringAndFragment(originalHref);
         }
+
+        public string Markup(string markdown, string sourceFileKey)
```

Comment:
yes, added it to system metadata when adding templateUtility.ResolveSourceRelativePath